src/MongoDB/Query.c

```diff
@@ -337,6 +337,8 @@ bool phongo_query_init(zval* return_value, zval* filter, zval* options)
 			phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT, "Expected \"modifiers\" option to be array, %s given", PHONGO_ZVAL_CLASS_OR_TYPE_NAME_P(modifiers));
 			return false;
 		}
+
+		php_error_docref(NULL, E_DEPRECATED, "The \"modifiers\" option is deprecated and will be removed in a future release");
```

Comment:
Note that there are a number of options below that are extracted from `modifiers` using one of the `_DEPRECATED` macros (e.g. `$maxScan`). We may want to change those to avoid creating double deprecation notices.